tests/integration-service/gitlab-integration-reporting.go

```diff
@@ -81,15 +81,27 @@ var _ = framework.IntegrationServiceSuiteDescribe("Gitlab Status Reporting of In
 		})
 
 		AfterAll(func() {
+			// first check if MR exists
+			_, _, err = f.AsKubeAdmin.CommonController.Gitlab.GetClient().MergeRequests.GetMergeRequest(projectID, mrID, nil)
+			if err != nil {
+				fmt.Printf("failed to get MR of IID %d in projectID %s, %v", mrID, projectID, err)
+			} else {
+				Expect(f.AsKubeAdmin.CommonController.Gitlab.CloseMergeRequest(projectID, mrID)).NotTo(HaveOccurred())
+
+			}
+
```

Comment:
I think this same code is being called within the [CloseMergeRequest() function](https://github.com/konflux-ci/e2e-tests/blob/main/pkg/clients/gitlab/git.go#L108-L112), with the above change, we're doing the same thing twice. Am I missing something?